index.html

```diff
@@ -0,0 +1,151 @@
+        </div>
+
+        <div id="favorite-feature">
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Front-end Projects">
+                    Front-end Projects
+                </label>
+            </div>
+
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Back-end Projects">
+                    Back-end Projects
+                </label>
+            </div>
+
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Data Visualization">
+                    Data Visualization
+                </label>
+            </div>
+
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Challenges">
+                    Challenges
+                </label>
+            </div>
+    
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Open Source Community">
+                    Open Source Community
+                </label>
+            </div>
+    
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Gitter help rooms">
+                    Gitter help rooms
+                </label>
+            </div>
+    
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Videos">
+                    Videos
+                </label>
+            </div>
+    
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="City Meetups">
+                    City Meetups
+                </label>
+            </div>
+    
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Wiki">
+                    Wiki
+                </label>
+            </div>
+          
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Forum">
+                    Forum
+                </label>
+            </div>
+            
+            <div class="input-element"> 
+                <label>
+                    <input type="checkbox" name="ideas" value="Additional Courses">
+                    Additional Courses
+                </label>
+            </div>
+        </div>
+        <div>
+            <p>ANY COMMENTS OR SUGGESTIONS?</p>
+            <textarea placeholder="ENTER YOUR COMMENT HERE..." name="textarea" id="textarea" cols="30" rows="10"></textarea>
+        </div>
+          <div>
+            <button id="submit" type="submit" class="submit-button">Submit</button>
+          </div>
+        
```

Comment:
Please, do not forget to add linters for your project, you can find the documentation by clicking [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

Reply:
Same here I suggest that you move this comment to the final review message as it is not related to this scope of code